frontend/src/metabase/components/Popover/Popover.css

```diff
@@ -177,7 +177,8 @@
   z-index: 1;
 }
 
-.PopoverBody--withBackground .PopoverFooterWhenIsNotEqualOps {
+.PopoverBody--withBackground .PopoverFooterWhenIsNotEqualOps,
+.popover .PopoverFooterWhenIsNotEqualOps {
```

Comment:
`.PopoverBody--withBackground` is what's I believed to be always set in the previous `Popover`. 
https://github.com/metabase/metabase/blob/60502ec8acc8de8eecf07a37822437c9f807d95a/frontend/src/metabase/components/Popover/Popover.jsx#L145

Since `hasBackground` has a default value of `true`.

But since `PopoverBody--withBackground` is only set in the previous popover. I need to add a selector for the new popover as well.

## Without this selector
![image](https://user-images.githubusercontent.com/1937582/165933182-b76b43b9-f827-4b6f-889a-56fdc26d8d10.png)

## Without this selector
![image](https://user-images.githubusercontent.com/1937582/165933145-159261b3-1a21-4bd5-8b67-1ce2bb33160c.png)